frontend/src/components/Pages/Todo/TodoItem.jsx

```diff
@@ -1,17 +1,41 @@
-import React from 'react';
-import { renderItems } from './TodoList';
+import React, { useState } from "react";
+import { renderItems } from "./TodoList";
+import Checkbox from "@mui/material/Checkbox";
+import FormGroup from "@mui/material/FormGroup";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import "./todoitem.css";
+
+const label = { inputProps: { "aria-label": "Checkbox demo" } };
 
 function TodoItem({ itemData }) {
-    return (
-        <li>
-            <p>{itemData.content}</p>
-            {itemData.items && itemData.items.length !== 0 &&
-                (<ul>
-                    {renderItems(itemData.items)}
-                </ul>)
-            }
-        </li>
-    );
+  const [checked, setChecked] = useState(false);
+
```

Comment:
For the initial state, use `itemData.isComplete || false`, rather than just false :)

Reply:
Sounds good!